README.rst

```diff
@@ -36,12 +36,22 @@ produced by JWST MIRI), along with 1D spectra extracted from the cube.
 **Imviz** provides visualization and quick-look analysis for 2D astronomical
 images.
 
+This tool is designed with instrument modes from the James Webb Space Telescope (JWST) in mind, but 
+the tool should be flexible enough to read in data from many astronomical telescopes.  The documentation provides a complete `table of all supported modes <https://jdaviz.readthedocs.io/en/latest/#jdaviz_instrument_table>`_.
```

Comment:
Actually, I think direct URL ref is different from Sphinx ref. See https://jdaviz--698.org.readthedocs.build/en/698/#jwst-instrument-modes-in-jdaviz

```suggestion
the tool should be flexible enough to read in data from many astronomical telescopes.  The documentation provides a complete `table of all supported modes <https://jdaviz.readthedocs.io/en/latest/#jwst-instrument-modes-in-jdaviz>`_.
```